components/snowflake/snowflake.app.mjs

```diff
@@ -1,9 +1,35 @@
-const { promisify } = require("util");
-const snowflake = require("snowflake-sdk");
+import snowflake from "snowflake-sdk";
+import { promisify } from "util";
 
-module.exports = {
+export default {
   app: "snowflake",
   type: "app",
+  propDefinitions: {
+    tableName: {
+      type: "string",
+      label: "Table Name",
+      async options(context) {
+        const { page } = context;
+        if (page !== 0) return [];
```

Comment:
Please correct me if I am confused. With `if (page !== 0) return []` wouldn't second page always be empty? If we are not going to use `page` variable, can we just delete lines 12 and 13 and return `options` like it is done for `columns` prop?